distribution/docs/src/main/resources/_contents/_draft-platform-contents/country-code-converter-contents.adoc

```diff
@@ -0,0 +1,32 @@
+==== Country Code Converter
+
+In the Platform Application, you have the option to install a feature called `platform-country-local`.
+This feature provides an OSGi service to convert country codes into various formats by performing a local lookup.
+
+In order to utilize the FIPS 10-4 to ISO 3166-1 alpha 3 and vice versa country code conversions, the `FIPS 10-4 to ISO 3166-1 alpha 3 File Location` field in the feature's configuration must be entered with the file path of a properties file containing the FIPS 10-4 to ISO 3166-3 alpha 3 country code mappings. The feature is already configured by default upon installation, with the file location field pointing to a properties file containing country code mappings from http://www.opengeocode.org/. This file can be found at: ${branding-lowercase}.home/etc/fipsToIso3.properties, and should only be modified in the case the country code mappings would ever change.
+
+The mapping file contains each mapping in the following format, where the country on the left side of the equals sign is FIPS and the country or countries on the right side of the equals sign is ISO, and a new line separating each mapping:
+
+* If there is a single FIPS 10-4 country that maps to a single ISO 3166-1 alpha 3 country: AB=ABC
+
+* If there is a single FIPS 10-4 country that maps to multiple ISO 3166-1 alpha 3 countries: ZZ=ZZZ,YYY
+
+===== Installing Country Code Converter
+
+The Country Code Converter is not installed by default with a standard installation.
+To install:
+
+. Navigate to the *${admin-console}*.
+. Select the *${ddf-platform}* application.
+. Select the *Features* tab.
+. Install the `platform-country-local` feature.
+
+===== Configuring Country Code Converter
```

Comment:
I wonder if we need to allow admins to change the path to the mapping file. It adds yet another configuration option that I'm not sure is really that useful. On top of that, if we keep it configurable, we will need to change the `PlatformMigratable` code since the file could be in a different directory or have a different name.